docs-src/guide-upgrading.scrbl

```diff
@@ -0,0 +1,56 @@
+#lang scribble/manual
+@(require "lib.rkt")
+
+@title[#:version reach-vers #:tag "guide-upgrading"]{Upgrading Reach}
+
+Upgrading Reach means (1) replacing your current @link["https://github.com/reach-sh/reach-lang/blob/master/reach"]{reach} script file with the latest script file, (2) replacing your current Reach Docker @emph{images} with the latest images, and (3) removing your current Reach Docker @emph{containers} built from the replaced images. The following directions assume that you installed the @link["https://github.com/reach-sh/reach-lang/blob/master/reach"]{reach} script file in @exec{~/reach}:
+
+@itemize[#:style 'ordered
+
+@item{
+  Download the latest reach script file:
+  @cmd{cd ~/reach}
+  @cmd{reach upgrade}
+  This command renames your current @link["https://github.com/reach-sh/reach-lang/blob/master/reach"]{reach} script file to something like @emph{reach.20040}, and downloads the lastest.
+}
+
+@item{
+  Remove @emph{all} Docker @emph{containers} (including those associated with Reach):
```

Comment:
I would say including those NOT associated with Reach, because that is more surprising